Hivemind/Assets/Scripts/Gathering.cs

```diff
@@ -13,8 +15,24 @@ private enum State
         Gathering,
         MovingToStorage,
     }
+
+    public enum Direction
```

Comment:
Indien deze enum alleen binnen deze classe blijft hou hem er lekker in, Als hij ook buiten deze klasse word gebruik haal hem er aub uit

Reply:
Hij wordt inderdaad alleen binnen deze klasse gebruikt